src/pages/billing/NewBillingProjectWizard/GCPBillingProjectWizard/GCPBillingProjectWizard.test.ts

```diff
@@ -44,12 +53,10 @@ const verifyChecked = item => expect(item).toBeChecked()
 const verifyUnchecked = item => expect(item).not.toBeChecked()
 
 const testStepActive = stepNumber => {
-  screen.getAllByRole('listitem').forEach((step, index) => {
+  screen.queryAllByTestId('Step').forEach((step, index) => {
```

Comment:
What was wrong with getting it by role?

Reply:
Ah, I think you changed it so these are no longer presented as being in an ordered list, and instead use region. An ordered list might make more sense here, since you are supposed to do them sequentially.

Reply:
Mostly, they stopped being `li` elements.
Also, this would get _all_ `li`s on the page, and I wanted to be able to use list elements in other components, if it made sense.
I could change it to get by `section` now, and I think that would be reasonable to assume wouldn't be used by any sub-components, but I changed the steps to section after I changed the test query (they were just `div`s for a while).

I'll go back and change that to querying by section, now that that is an option.

Reply:
Yes, using a list (I was say `ol` instead of `ul`, but either the `ul` in prod works well) makes much more sense when using a screen reader-- it tells you the number of steps, which step you are in, and the use of `aria-current` within it makes more sense.

Reply:
I put it back to a list, after we established it's completely fine to nest lists in html.